src/common/blkdev.cc

```diff
@@ -681,7 +681,7 @@ static int block_device_run_smartctl(const string& devname, int timeout,
     "smartctl",
     "-a",
     //"-x",
-    "--json",
+    "--json=o",
```

Comment:
it took me a while to find out what "o" was exactly for. 

just for posterity. "=o" is for 

see https://github.com/mirror/smartmontools/blob/ef9000ed70707ed25a9bd4beb46fd7d2e2ddbeb6/smartctl.cpp#L1106

https://github.com/mirror/smartmontools/blob/ef9000ed70707ed25a9bd4beb46fd7d2e2ddbeb6/smartctl.cpp#L1327

and

https://github.com/mirror/smartmontools/blob/ef9000ed70707ed25a9bd4beb46fd7d2e2ddbeb6/smartctl.cpp#L1299